boards/nrf51dk/src/main.rs

```diff
@@ -139,6 +140,7 @@ impl kernel::Platform for Platform {
             9 => f(Some(self.button)),
             14 => f(Some(self.rng)),
             17 => f(Some(self.aes)),
+            33 => f(Some(self.radio)),
```

Comment:
If you're adding a new syscall number, make sure you document it in [doc/Syscalls.md](https://github.com/helena-project/tock/blob/master/doc/Syscalls.md) (at the end)